phpstan-baseline.neon

```diff
@@ -175,6 +125,26 @@ parameters:
 			count: 2
 			path: tests/Language/VisitorTest.php
 
+		-
+			message: "#^Call to static method PHPUnit\\\\Framework\\\\Assert\\:\\:assertSame\\(\\) with Closure\\(\\)\\: array\\<int, array\\{message\\: string, locations\\?\\: array\\<int, array\\{line\\: int, column\\: int\\}\\>, path\\?\\: array\\<int, int\\|string\\>, extensions\\?\\: array\\<string, mixed\\>\\}\\> and array\\{'GraphQL\\\\\\\\Tests\\\\\\\\Server\\\\\\\\ServerConfigTest', 'handleError'\\} will always evaluate to false\\.$#"
```

Comment:
I considered these false positive

Reply:
Looks like it, I fixed them regardless by typing the variables at declaration.

Reply:
Nice